plugin/src/test/java/com/h3xstream/findsecbugs/cookie/CookieFlagsDetectorTest.java

```diff
@@ -84,18 +84,15 @@ public void avoidSecureFlagBasicFalsePositive() throws Exception {
         }
 
         // Advanced checks when multiple cookies are set
-        verify(reporter,never()).doReportBug(
-                bugDefinition()
-                        .bugType("INSECURE_COOKIE")
-                        .inClass("InsecureCookieSamples").inMethod("multipleCookies").atLine(68)
-                        .build()
-        );
-        verify(reporter,never()).doReportBug(
-                bugDefinition()
-                        .bugType("HTTPONLY_COOKIE")
-                        .inClass("HttpOnlyCookieSamples").inMethod("multipleCookies").atLine(79)
-                        .build()
-        );
+        List<Integer> lines = Arrays.asList(new Integer[] { 68, 79, 88 });
+        for (int line : lines) {
+            verify(reporter, never()).doReportBug(
```

Comment:
Avoid using **never()** with the property **line** set. The broken tests could be badly fixed with new lines and stilll pass.

Most tests are using this pattern:
 - Validate true positive first. (with the lines specify)
 - Validate the count of true positive (`verify(reporter,times(X)).bugType("BUG_X").inClass("Class").inMethod("method").build()`)